web/src/app/admin/AdminFieldComponents.tsx

```diff
@@ -60,11 +63,29 @@ export function StringInput(props: InputProps): JSX.Element {
       </InputAdornment>
     )
   }
+
+  if (type === 'tel') {
+    return (
+      <TelTextField
+        InputProps={{
+          startAdornment: (
+            <InputAdornment position='start' style={{ marginBottom: '0.1em' }}>
+              +
+            </InputAdornment>
+          ),
+        }}
```

Comment:
I think it would be worth having this be built into `TelTextField`, rather than passed in with InputProps, since we will always be handling the `+` within it now.

Like how you added in the end adornment